### ISSUE TITLE
Qtracker

### ISSUE THREAD

**Issue by TDNeo** [garncarz/nogamespy-vietcong#21]
Hey Ondra,

hope you are fine? Long time ago you were helpingh me with Qtracker ...

You may still care about that?

If so i have some question about QTracker: Some month ago it starts with verly long time to create a vc-server listing. Actually it does not work at all. 

My server is avtually not even listet inside the game but can be found in the QTracker favourite list.

You may help?

Best regards
=TD=Neo

**Comment by garncarz** [comment]
Hello Neo, 👋 

do you mean by this that your server is visible on Qtracker (e.g. here: http://www.qtracker.com/server_list_details.php?game=vietcong) and not visible on https://vietcong1.eu/game/ at the same time? What is your IP address (and port of the game)?

**Comment by TDNeo** [comment]
Hello Ondra,

 

thank you for your answer and help.

 

IP and port of my server: 194.97.164.252:5425

 

My server is still online and i can join my server by direct connection via vc:

 



 

It can also be seen in QTRacker:

 



 

Unfortunatelly my QTRacker does not longer find other servers:

 



 

Sometimes other servers are listed here after a very long time. Don’t know why this effect happens since a few month.

 

Following this links:

 

http://www.qtracker.com/server_list_details.php?game=vietcong) 

 

my server ist listed:

 



 

But with another port 15427. I don’t know, why this port is used here. In the game port 5425 is used.

 

 

Under this link:

 

https://vietcong1.eu/game/

 

my server is also not listed. 

 

 

Do you have any idea? Someone told me that country filters could be responsible for this effect …

 

Best 

=TD=Neo

 

Von: Ondřej Garncarz <notifications@github.com> 
Gesendet: Montag, 25. November 2019 20:31
An: garncarz/nogamespy-vietcong <nogamespy-vietcong@noreply.github.com>
Cc: TDNeo <TDNeo@gmx.de>; Author <author@noreply.github.com>
Betreff: Re: [garncarz/nogamespy-vietcong] Qtracker (#21)

 

Hello Neo, 👋

do you mean by this that your server is visible on Qtracker (e.g. here: http://www.qtracker.com/server_list_details.php?game=vietcong) and not visible on https://vietcong1.eu/game/ at the same time? What is your IP address (and port of the game)?

**Comment by garncarz** [comment]
@TDNeo Sorry for the delay, I thought I would have some time for this on this weekend, but I didn't manage. 🙂 Is that server of yours online non-stop, or what time usually?

**Comment by TDNeo** [comment]
Hey Ondra, no problem. I am glad that you do care :) My server is online anytime. It’s hosted @ 4NetPlayers.

 

Tnx in advance

=TD=Neo

 

Von: Ondřej Garncarz <notifications@github.com> 
Gesendet: Sonntag, 1. Dezember 2019 19:13
An: garncarz/nogamespy-vietcong <nogamespy-vietcong@noreply.github.com>
Cc: TDNeo <TDNeo@gmx.de>; Mention <mention@noreply.github.com>
Betreff: Re: [garncarz/nogamespy-vietcong] Qtracker (#21)

 

@TDNeo <https://github.com/TDNeo>  Sorry for the delay, I thought I would have some time for this on this weekend, but I didn't manage. 🙂 Is that server of yours online non-stop, or what time usually?

**Comment by garncarz** [comment]
@TDNeo Sorry for much longer delay now! I'm quite busy lately, new job, Christmas... 🎄 🙂 

✔️ I can connect to `194.97.164.252:5425` in the game.
✔️ Server can be registered manually:
```py
>>> from nogamespy import tasks
>>> tasks.register('194.97.164.252', 15425)
[2019-12-30 13:48:09,702][DEBUG] nogamespy.tasks tasks.py:register:153 | Trying to register 194.97.164.252:15425...
True
```
✔️ So now it can be accessed on https://vietcong1.eu/game/server/194.97.164.252.
❔ Not sure why it wasn't registered automatically before. I have to keep an eye on it. 👀

**Comment by TDNeo** [comment]
Hey Ondra,

 

thank you very much for yourt effords. Congratulation to new job. Hope you feel well with it ….

 

May I ask you another question?: Why is it, that there are no more servers listed in Qtracker?

 



 

 

All I can see is MyServer here:

 



 

Best regards

=TD=Neo

 

 

 

Von: Ondřej Garncarz <notifications@github.com> 
Gesendet: Montag, 30. Dezember 2019 14:57
An: garncarz/nogamespy-vietcong <nogamespy-vietcong@noreply.github.com>
Cc: TDNeo <TDNeo@gmx.de>; Mention <mention@noreply.github.com>
Betreff: Re: [garncarz/nogamespy-vietcong] Qtracker (#21)

 

@TDNeo <https://github.com/TDNeo>  Sorry for much longer delay now! I'm quite busy lately, new job, Christmas... 🎄 🙂

✔️ I can connect to 194.97.164.252:5425 in the game.
✔️ Server can be registered manually:

>>> from nogamespy import tasks
 
>>> tasks.register('194.97.164.252', 15425)
 
[2019-12-30 13:48:09,702][DEBUG] nogamespy.tasks tasks.py:register:153 | Trying to register 194.97.164.252:15425...
 
True

✔️ So now it can be accessed on https://vietcong1.eu/game/server/194.97.164.252.
❔ Not sure why it wasn't registered automatically before. I have to keep an eye on it. 👀

**Comment by garncarz** [comment]
@TDNeo Thanks. 🙂 
I can see some servers on http://www.qtracker.com/server_list_details.php?game=vietcong, so it seems Qtracker itself is working fine. What do you mean by `MyServer`, that's some Qtracker app?
But whatever that is, maybe it's because one game server is acting weirdly (returning value:key pairs instead of key:value), see #22 – quite a coincidence, it seems it started yesterday.

**Comment by TDNeo** [comment]
Hey Ondra,

 

„MyServer“ is just a favourite folder. Once I copied my server to this folder. That is, why I can seen my server there:

 



 

 

Did you changed anything???

 

Now it is working:

 



 

:)

 

 

 

 

 

Von: Ondřej Garncarz <notifications@github.com> 
Gesendet: Donnerstag, 2. Januar 2020 18:50
An: garncarz/nogamespy-vietcong <nogamespy-vietcong@noreply.github.com>
Cc: TDNeo <TDNeo@gmx.de>; Mention <mention@noreply.github.com>
Betreff: Re: [garncarz/nogamespy-vietcong] Qtracker (#21)

 

@TDNeo <https://github.com/TDNeo>  Thanks. 🙂
I can see some servers on http://www.qtracker.com/server_list_details.php?game=vietcong, so it seems Qtracker itself is working fine. What do you mean by MyServer, that's some Qtracker app?
But whatever that is, maybe it's because one game server is acting weirdly (returning value:key pairs instead of key:value), see #22 <https://github.com/garncarz/nogamespy-vietcong/issues/22>  – quite a coincidence, it seems it started yesterday.

**Comment by garncarz** [comment]
No, no, I haven't changed anything during today. If it works, that's the main thing. 🙂  So I'll close this issue, but feel free to reopen it (or create a new one) if something errs again. Thanks for the feedback already provided!

**Comment by sonicpp** [comment]
Hmm, I can see Neo's server on vietcong.tk web serverlist, while it is not listed at vietcong1.eu web serverlist...

**Comment by garncarz** [comment]
I'm experimenting with another approach with Celery workers / Redis (#24), but Redis has higher memory usage that way and it's reaching a limit on my VPS, so fetching new servers can be a bit unstable these days. I need to limit number of Redis keys.

But thanks for reporting this. 👍
